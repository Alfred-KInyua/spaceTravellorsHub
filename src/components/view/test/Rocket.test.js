import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../app/Store';
import RocketList from '../rockets/Rocket';

describe('Testing RocketList component', () => {
  it('Testing with snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <RocketList />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
