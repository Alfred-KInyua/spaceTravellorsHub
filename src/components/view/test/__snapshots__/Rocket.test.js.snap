// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing RocketList component Testing with snapshot 1`] = `
<div
  className="d-flex flex-row bd-highlight mb-3 justify-content-center"
>
  <div
    style={
      Object {
        "width": "400px",
      }
    }
  >
    <img
      alt="Responsive"
      className="p-3"
      style={
        Object {
          "width": "400px",
        }
      }
    />
  </div>
  <div
    className="mt-3"
  >
    <div>
      <h2 />
    </div>
    <div>
       
    </div>
    <div>
       
      <button
        className="btn btn-primary"
        onClick={[Function]}
        type="button"
      >
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;
